### DISCUSSION TITLE
Custom Domain / should redirect to plausible.io

### DISCUSSION THREAD

**Discussion by bennihtm** [plausible/analytics#776] [Anything Else]
It would be nice to have the custom domain redirect to plausible.io if it doesn't access the script directly.
For example if I visit stats.domain.com/js/index.js it serves me the plausible script but if I visit stats.domain.com it redirects my to plausible.io.
Just to not show a 404 Error, and be more open on what tool is used.

**Comment by metmarkosaric** [comment]
thanks @benjaminwolkchen! would be good to do for sure! there's a similar request already at #649
